ecosystem/ecosystem.html

```diff
@@ -146,4 +152,8 @@ <h1>
     $(".nav-link[data-category=" + category + "]").addClass('selected');
   }
 </script>
+
 <script star-count-call-date="ecosystemStarCountCallDate" star-count-data="ecosystemStarCountData" src="{{ site.baseurl }}/assets/github-stars.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
+<script list-id="ecosystem-index-cards" display-count="100" pagination="false" src="{{ site.baseurl }}/assets/filter-hub-tags.js"></script>
+<script src="{{ site.baseurl }}/assets/hub-sort.js"></script>
```

Comment:
You don't need this hub-sort script since the sorting will be coming from filter-hub-tags.